schema-admin.graphql

```diff
@@ -521,18 +539,17 @@ input ImportApprovedCuratedCorpusItemInput {
     """
     imageUrl: Url!
     """
-    A topic this story best fits in. The value will be `imported` for migrated items that don't have a topic set.
+    A topic this story best fits in. The value will be `IMPORTED` for migrated items that don't have a topic set.
     """
     topic: String!
     """
-    Whether this story is a Pocket Collection.
+    The source of the corpus item.
     """
-    isCollection: Boolean
+    source: CorpusItemSource!
```

Comment:
one could argue this mutation implies `BACKFILL`, but i don't have a big problem with this being an input.

Reply:
it might be good to have it explicit, so if data engineering wants to use this mutation later  for non new-tab related loads, they can add their custom enum directly to the source